docs/how-to-guides/upgrade-bootstrap-kubelet.md

```diff
@@ -0,0 +1,57 @@
+# How to upgrade bootstrap Kubelet
+
+## Contents
+
+- [Introduction](#introduction)
+- [Steps](#steps)
+  - [Drain the node](#drain-the-node)
+  - [Find out the IP and SSH](#find-out-the-ip-and-ssh)
+  - [On the node](#on-the-node)
+- [Caveats](#caveats)
+
+## Introduction
+
+Bootstrap Kubelet is the Kubelet running on the node as a systemd service outside Kubernetes cluster's control. Unlike most cluster components, `lokoctl` cannot update it(which may change in the future). This document enlists the manual steps involved in updating the bootstrap Kubelet.
+
+## Steps
+
+Apply the following steps to all the cluster nodes one at a time.
+
+### Drain the node
+
+> **Caution:** If you are using a local directory as a storage for a workload, it will be disturbed by this operation. So move the workload to another node and let the application replicate the data. If the application does not support data replication across instances, then expect downtime.
+
+```bash
+kubectl drain --ignore-daemonsets <node name>
+```
+
+### Find out the IP and SSH
+
+Find the IP of the node by visiting the Cloud provider dashboard.
+
+```
+ssh core@<IP Address>
+```
+
+### On the node
+
+Run the following commands.
+
+> **NOTE**: Export the correct and latest Kubernetes version, before proceeding to other commands.
```

Comment:
```suggestion
>NOTE: Before proceeding to other commands, set the `latest_kube` variable to the latest Kubernetes version.
```

I'd avoid squeezing a comment between "Run the following commands:" and the commands. Maybe this can be an inline shell comment.

Reply:
What does "latest" mean? How should the user figure out which version to use?

Reply:
> What does "latest" mean? How should the user figure out which version to use?

That's a good question. There should be a way for user to know what is the latest kubernetes in `lokoctl`. Since this is upgrading the self-hosted kubelet it is safe to assume that user has already upgraded the control plane, so here it only way to find the lates kuberenetes version is by running `kubectl version` and see what version the server is at.

I think we can say somewhere in the doc that user should upgrade the control plane already. And then ask them to figure out the latest kubernetes version to upgrade kubelet to would be found by running following command:

```bash
kubectl version --short | grep ^Server | awk '{print $3}'
```